model/transformers/dffml_model_transformers/classification/classification_model.py

```diff
@@ -0,0 +1,482 @@
+    )
+
+    cache_dir: str = field(
+        "Directory to store the pre-trained models downloaded from s3",
+        default=str(pathlib.Path("~", ".cache", "dffml", "transformers")),
+    )
+
+    doc_stride: int = field(
+        "When splitting up a long document into chunks, how much stride to take between chunks.",
+        default=128,
+    )
+    optimizer_name: str = field(
+        'Name of a Tensorflow optimizer among "adadelta, adagrad, adam, adamax, ftrl, nadam, rmsprop, sgd, adamw"',
+        default="adam",
+    )
+    loss_name: str = field(
+        "Name of a Tensorflow loss. For the list see: https://www.tensorflow.org/api_docs/python/tf/keras/losses",
+        default="SparseCategoricalCrossentropy",
+    )
+    gpus: str = field(
+        "List of gpu devices. If only one, switch to single gpu strategy, if None takes all availabel gpus",
+        default="0",
+    )
+    no_cuda: bool = field("Avoid using CUDA when available", default=False)
+
+    end_lr: float = field("End learning rate for optimizer", default=0)
+    debug: bool = field(
+        "Activate the trace to record computation graphs and profiling information",
+        default=False,
+    )
+
+    output_dir: str = field(
+        "The output directory where the model predictions and checkpoints will be written.",
+        default=str(pathlib.Path("~", ".cache", "dffml", "transformers")),
+    )
+    overwrite_output_dir: bool = field(
+        "Overwrite the content of the output directory.Use this to continue training if output_dir points to a checkpoint directory.",
+        default=False,
+    )
+
+    evaluate_during_training: bool = field(
+        "Run evaluation during training at each logging step.", default=False,
+    )
+
+    per_device_train_batch_size: int = field(
+        "Batch size per GPU/TPU core/CPU for training.", default=8,
+    )
+    per_device_eval_batch_size: int = field(
+        "Batch size per GPU/TPU core/CPU for evaluation.", default=8,
+    )
+
+    gradient_accumulation_steps: int = field(
+        "Number of updates steps to accumulate before performing a backward/update pass.",
+        default=1,
+    )
+
+    learning_rate: float = field(
+        "The initial learning rate for Adam.", default=5e-5,
+    )
+    weight_decay: float = field(
+        "Weight decay if we apply some.", default=0.0,
+    )
+    adam_epsilon: float = field(
+        "Epsilon for Adam optimizer.", default=1e-8,
+    )
+    max_grad_norm: float = field(
+        "Max gradient norm.", default=1.0,
+    )
+
+    num_train_epochs: float = field(
+        "Total number of training epochs to perform.", default=1,
+    )
+    max_steps: int = field(
+        "If > 0: set total number of training steps to perform. Override num_train_epochs.",
+        default=-1,
+    )
+    warmup_steps: int = field(
+        "Linear warmup over warmup_steps.", default=0,
+    )
+
+    logging_dir: str = field(
```

Comment:
This looks unused, let's remove it (and from the other model if it's unused there)

Reply:
This is used internally by model for logging, so we can't remove it.